nx/lib/nx/pytorch_backend.ex

```diff
@@ -0,0 +1,177 @@
+defmodule Nx.PytorchBackend do
+  @behaviour Nx.Backend
+
+  defstruct [:ref]
+
+  alias Nx.Pytorch.NIF
+
+  import Nx.Shared
+
+  funs =
+    Nx.Backend.behaviour_info(:callbacks) --
+      [
+        tensor: 1,
+        iota: 2,
+        eye: 1,
+        random_uniform: 3,
+        random_normal: 3,
+        reshape: 3,
+        squeeze: 3,
+        add: 3,
+        dot: 5,
+        cholesky: 2,
+        from_binary: 3,
+        to_binary: 2,
+        backend_deallocate: 1,
+        inspect: 2
+      ]
+
+  for {fun, arity} <- funs do
+    args = Macro.generate_arguments(arity, __MODULE__)
+
+    def unquote(fun)(unquote_splicing(args)) do
+      raise "#{unquote(fun)}() is not supported"
+    end
+  end
+
+  ## Creation
+
+  @impl true
+  def tensor(tensor) do
```

Comment:
Please rebase because this callback no longer exists. :)

Reply:
But I see it here: 
https://github.com/elixir-nx/nx/blob/e5d51f4ad04e7cd6612289093bc0dfbca99e9f84/nx/lib/nx/backend.ex#L46

And still don't get what's it's purpose:)

Reply:
That was a left-over. I have removed it!